crates/native/Cargo.toml

```diff
@@ -8,6 +8,11 @@ publish = false
 [lib]
 crate-type = ["cdylib"]
 
+[features]
+default = ["renderer_cpp_api"]
+renderer_cpp_api = ["dep:cxx-build"]
+renderer_c_api = ["dep:cc"]
```

Comment:
@evdokimovs why do we have this features, which are, indeed, not additive? This complicates linting, docs, and all the stuff.

Let's strip the features out and use the `target_os =` conditions.